core/build.gradle

```diff
@@ -21,6 +21,7 @@ managedDependencies {
         compile "io.netty:$it"
     }
     compile 'io.netty:netty-transport-native-epoll:linux-x86_64'
+    compile 'io.netty:netty-transport-native-kqueue:osx-x86_64'
```

Comment:
I agree with you that nobody uses Mac servers in production and thus it's pretty much likely that it's wasteful to pull it in. Let me leave this up to you and @imasahiro since I don't have any strong opinion on this.